src/Tabs/Tabs.css

```diff
@@ -1,14 +1,3 @@
-.tab-group {
-  display: flex;
-  flex-direction: column;
-
-  padding: 1rem;
-}
-
-.tab-group--vertical {
-  flex-direction: row;
-}
-
 .tab-group__tab-list {
```

Comment:
tab list can be replaced with a Container component if Container can take a general style prop